client/src/components/HomeDetails.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import { Tooltip } from '@nextui-org/react';
+import { AiOutlineWifi } from 'react-icons/ai';
+import { BiSolidParking } from 'react-icons/bi';
+
+export const HomeDetails = () => {
```

Comment:
- [ ] When I click on a home card to see its details, it works wonderfully. However if I click on the "View Home" button, it does not work:

![image](https://github.com/peterdtitan/BnB-homes/assets/57421823/fb9b426b-87dc-4bcd-82aa-902fe286ed2b)

Kindly, make it so the "View home" button takes me to the house's details.

That way, your UX will be improved. 👍

Reply:
Thanks for the suggestion. Done ✅ ✅